src/settings-ui/Settings.UI.Library/HostsProperties.cs

```diff
@@ -13,6 +14,7 @@ public class HostsProperties
         public bool ShowStartupWarning { get; set; }
 
         [JsonConverter(typeof(BoolPropertyJsonConverter))]
+        [CmdConfigureIgnore]
         public bool LaunchAdministrator { get; set; }
```

Comment:
Why is this one being ignored, it's an actual setting, right? That you can configure in Settings. Whether you want to launch hosts as an administrator or not.